br/pkg/utils/retry_test.go

```diff
@@ -5,55 +5,58 @@ import (
 	"fmt"
 	"io"
 	"net"
-	"testing"
 
 	"github.com/go-sql-driver/mysql"
+	. "github.com/pingcap/check"
```

Comment:
currently we use `"github.com/stretchr/testify/require"` instead of `"github.com/pingcap/check"`, do not roll back it.